mix.exs

```diff
@@ -33,7 +33,7 @@ defmodule ClusterEC2.Mixfile do
       {:sweet_xml, "~> 0.6"},
       {:hackney, "~> 1.8"},
       {:poison, ">= 1.0.0"},
-      {:tesla, "~> 1.0.0"},
+      {:tesla, "~> 1.1.0"},
```

Comment:
Please change to "~> 1.0" to allow minor version compatibility (besides 1.1.X, allow also 1.2.X, etc.; but not 2.X.X).
See https://hexdocs.pm/elixir/Version.html